pyvista/core/filters/structured_grid.py

```diff
@@ -3,7 +3,7 @@
 
 import pyvista
 from pyvista import abstract_class, _vtk
-from pyvista.core.filters import _get_output
+from pyvista.core.filters import _get_output, _update_alg
```

Comment:
This is highly suspicious because this is the only change in the file.

Reply:
Agreed. This PR no longer touches this file.